packages/sdk/src/express-authentication-middleware/express.ts

```diff
@@ -1,6 +1,7 @@
 import type { NextFunction, Request, Response } from 'express';
 
-import { jwt } from '@lit-protocol/vincent-sdk';
```

Comment:
💖 Good catch, I just spotted this tonight and was going to open a PR to fix that reference 🍭